internal/utils.go

```diff
@@ -492,6 +492,11 @@ func buildPolicyTemplate(
 		"severity":          policyConf.Severity,
 	}
 
+	if configPolicyOptionsOverrides.CustomMessage.Compliant != "" ||
+		configPolicyOptionsOverrides.CustomMessage.NonCompliant != "" {
+		policySpec["customMessage"] = configPolicyOptionsOverrides.CustomMessage
+	}
```

Comment:
I think it'd make sense to handle this similar to `evaluationInterval` where `Compliant` and `NonCompliant` are handled separately. I could see a case where a user wants to have a default brief message for everything and then override just `NonCompliant` for one policy (or vice versa).

(and probably move this block adjacent to it since they're both similar settings--though I'm not entirely sure how the `policyTemplate` wound up in the middle of this function...)

Reply:
I think due to the omitempty in the YAML tag on the struct, it ends up doing the same thing but let's make it consistent with evaluation interval.